src/app/login-page/login-page.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Login } from '../models/login.interface';
+import { AuthenticationService } from '../shared/services/authentication.service';
+
+@Component({
+  selector: 'login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss']
+})
+export class LoginPageComponent implements OnInit{
+  public loginForm: FormGroup;
+  public authKey = 'Auth_Key';
```

Comment:
incapsulate this key in the service, it should be private to it. Only auth service should know where it stores data, it's not a responsibility of this component